CMSgov/design-system
# [WNMGDS-693] Allow hiding doc site pages

### Summary
- Adds `hideSection` flag to hide KSS sections or pages

### How to test
- Add `@hide-section` to a KSS page and ensure the entire page is removed from navigation, and no html files related to that page are generated (including example pages)
- Add `@hide-section` to a KSS subsection and ensure that section is removed
- Add `hideSection` to a markdown file and ensure that section is removed
- Do all the above steps on a Child DS as well
